namespaces/collectd.yml

```diff
@@ -51,29 +66,54 @@ namespace:
           TODO
 
     - name: ps_vm
-      type: integer
+      type: group
```

Comment:
This is the way the nest plugin creates the record.
If there is only 1 value in the record sent by collectd the dsname is ["values"]
If there is more than 1 then the dsname will be something like ["read","write"] and other.